pkg/framework/api/interface.go

```diff
@@ -843,6 +843,8 @@ type ScheduleUnit interface {
 	UpdatePod(pod *QueuedPodInfo) error
 	// DeletePod deletes QueuedPodInfo from the Unit
 	DeletePod(pod *QueuedPodInfo) error
+	// MergePods merges pods into the Unit, will skill pod already existed in the Unit
+	MergePods(pods ...*QueuedPodInfo) error
```

Comment:
rename to `AddPodsIfNotPresent` ? please update the function comment too

Reply:
good idea.